WooCommerce/Classes/ViewRelated/Dashboard/Onboarding/StoreOnboardingView.swift

```diff
@@ -4,25 +4,70 @@ import SwiftUI
 struct StoreOnboardingView: View {
     private let viewModel: StoreOnboardingViewModel
     private let taskTapped: (StoreOnboardingTask) -> Void
+    private let viewAllTapped: (() -> Void)?
```

Comment:
nit: any reasons for this action handler to be optional? is there any case we don't want to handle the "view all" action?

Reply:
`StoreOnboardingView` when presented in expanded mode will not have a "View All" button. So, I kept this as an optional. Please correct me if I am missing something.